.cirrus.yml

```diff
@@ -156,3 +157,10 @@ task:
         cd %ENGINE_PATH%/src
         python flutter/tools/gn --runtime-mode debug
         ninja -C out/host_debug
+    - name: build_and_test_web_windows_edge
+      test_web_engine_edge_script: |
+        cd %ENGINE_PATH%/src/flutter/web_sdk/web_engine_tester
+        pub get
```

Comment:
Is the reason for the `PATH` change only to be able to call `pub` instead of `%ENGINE_PATH%/src/out/host_debug_unopt/dart-sdk/bin/pub`? If so, I'm not sure if it's saving much.

Reply:
yes for now. As I mentioned in the previous comment the issue is: "# Cirrus is somehow not picking up the environment variables set in the VM image."